FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -211,8 +241,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <note>130 stories, indicated by numbers in the margins or with rubricated numbers in the text up to 134, 
                                 while several stories are omitted and the numbering being confused: no. 13 omitted, no. 42 omitted, 
                                 one miracle after no. 89 and before no. 90 has no number, no. 100 and no. 101 omitted, no 121 is omitted.</note>
-                            <note>Some stories on <locus target="93"></locus> are similar to the some passages in the Life of Cyrus as it is recorded in <ref type="mss"  target="BLorient701">Ms. BMorient701 (= Wright 277)</ref>.</note>
-                                <colophon xml:id="coloph1"><locus target="113vb" xml:lang="gez"/>፻፴፬፡ ዘተጽሕፈ፡ ተአምር፡ ዘማርያም። በእዴሁ፡ ለ<persName ref="PRS0000">አርሳንዮስ፡</persName> ሊቀ፡ ካህናት፡</colophon>
+                            <note>Some stories on <locus target="93"></locus> are similar to the some passages in the Life of Cyrus as it is recorded in 
+                                <ref type="mss"  target="BLorient701">Ms. BMorient701 (= Wright 277)</ref>.</note>
+                            <colophon xml:id="coloph1"><locus target="113vb" xml:lang="gez"/>፻፴፬፡ ዘተጽሕፈ፡ ተአምር፡ ዘማርያም። በእዴሁ፡ 
```

Comment:
`xml:lang="gez"` may not be the attribute of `locus `(locus = page number only); it can be the attribute of `colophon`

Reply:
Done.